databox/indexer/src/lib/pathUtils.ts

```diff
@@ -8,7 +8,7 @@ export function stripSlashes(path: string): string {
 
 export function splitPath(path: string): string[] {
     return path
-        .replace(/^\//, '')
         .split(/(?<!\\)\//)
+        .filter(x => x)     // remove empty segments (/a//b/c/ => [a,b,c])
```

Comment:
Why would you need to correct an `//` (empty collection)? It would hide another error I guess...

Reply:
ease concat (build path) from strings that may already start/end with "/". 
empty coll-name should be detected earlier, not because we build a path.